### PR TITLE
Add tailwind to Audit Logs

### PR DESCRIPTION
**Pull request by abergs**
- Added tailwind
- Moved Datetime to be first
- Removed ToLocalDatetime, it uses server side time, not relevant
- World war on formatting
- Updated OrgLogs with TW

### CODE REVIEW

**Comment by abergs** [comment]
Updated design

**Comment by abergs** [comment]
thank you @willmartian for TW support

File: src/AdminConsole/Pages/Organization/Log.cshtml [hunk 1]
```diff
@@ -6,38 +6,104 @@
...
-                </tr>
-            }
-            </tbody>
-        </table>
-    }
-    else
-    {
-        <p>This organization has no events from the past @Model.RetentionPeriod days.</p>
-    }
+    <div class="flow-root" asp-if="Model.Events.Any()">
```
Comment (jrmccannon) [inline src/AdminConsole/Pages/Organization/Log.cshtml:9]: Just curious: Is there a benefit to using `asp-if` over `@if (...)`?

Also should the paging section at the bottom be changed to `asp-if` to be consistent?

**Review by jrmccannon — APPROVED** [review]
Just had a small question. Other than that looks good!

**Review by jonashendrickx — APPROVED** [review]

File: src/AdminConsole/Pages/Organization/Log.cshtml (hunk 1 above)
Comment (abergs) [reply src/AdminConsole/Pages/Organization/Log.cshtml:9]: @jrmccannon The only benefit to using `asp-if` is to reduce nesting and visual noise. Yeah, I think paging could be changed as well.

**Pull request merged**